APMrover2/Parameters.h

```diff
@@ -341,6 +341,12 @@ class ParametersG2 {
 
     // pivot turn rate
     AP_Int16 pivot_turn_rate;
+
+    // pitch angle at 100% throttle
+    AP_Float bal_pitch_max;
+
+    // pitch/roll angle for crash check
+    AP_Float crash_angle;
```

Comment:
this either needs to be an AP_Int8, or is_zero() needs to be used for the float. I suggest using an AP_Int8, as 1 degree resolution is fine